examples/step-45/doc/intro.dox

```diff
@@ -52,18 +53,30 @@ GridTools::collect_periodic_faces(dof_handler,
                                   first_vector_components = <default value>);
 @endcode
 
-This call loops over all faces of the container dof_handler on the opposing
+This call loops over all faces of the container dof_handler on the periodic
 boundaries with boundary indicator @p b_id1 and @p b_id2,
-respecitvely. If $\text{vertices}_{1/2}$ are the vertices of $\text{face}_{1/2}$,
-it matches pairs of faces (and dofs) such that the difference between $\text{vertices}_2$
-and $matrix\cdot \text{vertices}_1+\text{offset}$ vanishes in every component apart from direction
-and stores the resulting pairs with associated data in @p matched_pairs. (See
+respectively. (You can assign these boundary indicators by hand after
+creating the coarse mesh, see
+@ref GlossBoundaryIndicator "Boundary indicator". Alternatively, you
+can also let many of the functions in namespace GridGenerator do this
+for if you specify the "colorize" flag; in that case, these functions
+will assign different boundary indicators to different parts of the
+boundary, with the details typically spelled out in the documentation
+of these functions.)
+
+Concretely, if $\text{vertices}_{1/2}$ are the vertices of two faces
+$\text{face}_{1/2}$, the then function call above will match pairs of
```

Comment:
~~then~~

Reply:
Actually, I mean `then the` :-)

Fixed now!